views/repo_list.pug

```diff
@@ -1,9 +1,36 @@
-h1
-  Github Repositories
+h1 Github Repositories
 
 .git-repo-areas
 
   .git-repo( ng-repeat="repo in globalState.user.repos()" ng-controller="ListItemCtrl" )
-    .repo-name
-      span( ng-bind="repo.full_name" )
-      a.link-out( ng-href="{{repo.html_url}}" target="_blank" )
+    .repo-item
+      .name
+        a( ng-bind="repo.name" ng-href="{{repo.html_url}}" target="_blank" )
+        span.private-tag( ng-show="repo.private" ) Private
+      .full-name( ng-show="repo.fork" ) Forked from _______
+      .description( ng-show="repo.description" ng-bind="repo.description" )
+      .footer
+        .language( ng-show="repo.language" )
+          span.circle( style="background-color: {{getLanguageColor(repo.language)}}" )
+          | {{repo.language}}
+        .stars-count( ng-show="repo.stargazers_count" )
+          i.fa.fa-star
+          | {{repo.stargazers_count}}
+        .forks-count( ng-show="repo.forks_count" )
+          i.fa.fa-code-fork
+          | {{repo.forks_count}}
+        .updated-at {{repo.updated_at | date:"'Updated at' dd MMM yyyy"}}
+      .notification-btn-container
+        a.notification-btn( ng-click="getNotifications()" data-toggle="dropdown" )
+          i.fa.fa-bell
```

Comment:
@badeot, Okay, its now updated, and also changed the flex into float instead. 👍  Let me know again if you